examples/platform/silabs/wifi/icd/ApplicationSleepManager.cpp

```diff
@@ -53,17 +53,17 @@ CHIP_ERROR ApplicationSleepManager::Init()
 void ApplicationSleepManager::OnCommissioningWindowOpened()
 {
     mIsCommissionningWindowOpen = true;
-#if (defined(SLI_SI91X_MCU_INTERFACE) && SLI_SI91X_MCU_INTERFACE == 1)
+#if (defined(SLI_SI917) && SLI_SI917 == 1)
```

Comment:
This "SLI_SI917" need to be replaced with "SLI_SI91X_MCU_INTERFACE" in other places also?

Reply:
"SLI_SI917" : this macro is enabled for 917 NCP and SoC boards.

Reply:
Are there any other places where we need to replace this macro (common code for NCP and SoC)?

Reply:
No @senthilku